poc/vdaf_prio3.py

```diff
@@ -230,10 +249,16 @@ def shard_with_joint_rand(Prio3, meas, nonce, seeds):
             0, k_leader_blind, leader_meas_share, nonce))
 
         # Generate the proof and shard it into proof shares.
-        prove_rand = Prio3.prove_rand(k_prove)
-        joint_rand = Prio3.joint_rand(
+        prove_rands = Prio3.prove_rands(k_prove)
+        joint_rands = Prio3.joint_rands(
             Prio3.joint_rand_seed(k_joint_rand_parts))
-        leader_proof_share = Prio3.Flp.prove(meas, prove_rand, joint_rand)
+        leader_proof_share = []
```

Comment:
```suggestion
        leader_proofs_share = []
```

Reply:
I think this is reflected in the latest version.